libs/orders/src/lib/components/order-list/order-edit-dialog.tsx

```diff
@@ -88,15 +87,24 @@ export const OrderEditDialog = ({
         <form onSubmit={handleSubmit(onSubmit)} data-testid="edit-order">
           <FormGroup label={t('Entry price')} labelFor="entryPrice">
             <Input
-              {...register('entryPrice', { required: t('Required') })}
-              id="entryPrice"
               type="text"
+              step={step}
```

Comment:
chore (blocking): so, inputs with type="text" cannot have steps, only input type="number". if you make it a text input, the step attribute becomes redundant.